### PR TITLE
Update set-up-phone-system-voicemail.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@roykuntz Adding section for Teams users for managing Voicemail and Teams calling features to make it easier to find. This was in response to some questions in SME channels.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-phone-system-voicemail.md [hunk 1]
```diff
@@ -113,6 +113,12 @@ Grant-CsOnlineVoicemailPolicy -PolicyName TranscriptionProfanityMaskingEnabled -
 > [!IMPORTANT]
 > The voicemail service in Office 365 caches voicemail policies and updates the cache every 4 hours. So, policy changes that you make can take up to 4 hours to be applied.
 
+## Help your users learn Teams voicemail features
+
+We have information for your users on managing their voicemail settings as well as other calling features in Teams:
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md]:
```suggestion
We have the following information for your users on managing their voicemail settings as well as other calling features in Teams:
```

File: Teams/set-up-phone-system-voicemail.md [hunk 2]
```diff
@@ -113,6 +113,12 @@ Grant-CsOnlineVoicemailPolicy -PolicyName TranscriptionProfanityMaskingEnabled -
...
+
+- [Manage your call settings in Teams](https://support.office.com/en-ie/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f): This article explains how to manage all end user Teams calling features.
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md]:
```suggestion
- [Manage your call settings in Teams](https://support.office.com/article/manage-your-call-settings-in-teams-456cb611-3477-496f-b31a-6ab752a7595f). This article explains how to manage all end-user Teams calling features. 
```

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**